content/docs/command-reference/commit.md

```diff
@@ -1,7 +1,7 @@
 # commit
 
-Record changes to DVC-tracked files in the <abbr>project</abbr>, by saving them
-to the <abbr>cache</abbr> and updating the `dvc.lock` or `.dvc` files.
+Stores changed changed files and directories tracked by DVC in the
+<abbr>cache</abbr>, and update the `dvc.lock` or `.dvc` files if needed.
```

Comment:
store ... update
stores ... updates

Reply:
> store ... update

Will fix (may no longer be relevant when I address your next https://github.com/iterative/dvc.org/pull/1989#pullrequestreview-542464568).